web_src/js/modules/tippy.js

```diff
@@ -3,11 +3,7 @@ import tippy from 'tippy.js';
 const visibleInstances = new Set();
 
 export function createTippy(target, opts = {}) {
-  const {role, content, onHide: optsOnHide, onDestroy: optsOnDestroy, onShow: optOnShow} = opts;
-  delete opts.onHide;
-  delete opts.onDestroy;
-  delete opts.onShow;
-
+  const {onHide, onShow, onDestroy, ...other} = opts;
```

Comment:
Would also destructure `role` and `content` here.

Reply:
Can even shorten it by placing the destructure in the function definition:

````js
export function createTippy(target, {onHide, onShow, onDestroy, role, content, ...other} = {})
````

Reply:
> Would also destructure `role` and `content` here.

IMO the code would be more complex for using "role" later. Could you try to improve it by your idea?

Reply:
> export function createTippy(target, {onHide, onShow, onDestroy, role, content, ...other} = {})

Well, personally I dislike such syntax. But if you prefer, no block from my side.

Reply:
Definitely prefer that, it's most readable to me. Don't have time right now, but I can do a few modifications later.